fe/model_utils.py

```diff
@@ -53,12 +54,24 @@ def is_hydrogen(i):
     return masses
 
 
-# def generate_openmm_topology(objs, host_coords, out_filename):
-def generate_openmm_topology(objs, coords, out_filename=None):
+def image_molecule(mol_coords, box):
+    """Given a set of coordinates for a single molecule and a box will return
+    the coordinates wrapped into the periodic box.
+    """
+    centroid = np.mean(mol_coords, axis=0)
+    box_diag = np.diag(box)
```

Comment:
nit: what do you think about `np.diagonal` instead here? I find it potentially confusing that `np.diag` works on both 1- and 2-d arrays (and does a different thing in each case).